optimum_benchmark/trackers/emissions.py

```diff
@@ -0,0 +1,24 @@
+from contextlib import contextmanager
+from logging import getLogger
+
+from codecarbon import EmissionsTracker as CodeCarbonEmissionsTracker
+
+LOGGER = getLogger("latency_tracker")
+
+
+class EmissionsTracker:
+    def __init__(self):
+        self.emissions: float = 0.0
+
+    @contextmanager
+    def track(self, interval=1, file_prefix=""):
+        emission_tracker = CodeCarbonEmissionsTracker(
+            measure_power_secs=interval,
+            output_file=f"{file_prefix}_emissions.csv",
```

Comment:
So we can retrieve any measure provided by CodeCarbon in this CSV file right?

Reply:
Yes exactly. All the CodecCarbon measures are logged primarily in this file. The tracker only returns the carbon emissions but it needs internet to infer it from energy and location. Should we use the offline tracker instead ?

Reply:
Does it automatically fall back to the offline tracker if there is no internet connection?

Reply:
They rather implement two trackers, one is [online](https://mlco2.github.io/codecarbon/usage.html#online-mode) which does some api calls to get info about carbon emission rates and one is [offline](https://mlco2.github.io/codecarbon/usage.html#offline-mode) with a non-default `country_iso_code` argument.

Reply:
Okay I see. What happens if we use the online tracker without internet access? An error is raised? If yes, we can catch it in a try block and run the offline tracker in the except one.

Reply:
what about the `country_iso_code` argument ? I don't think you can get location without internet access

Reply:
Ah sorry I missed that bit. So the user has to provide this `country_iso_code` right?

Reply:
yes, should we use something like an environment variable to handle it ? instead of having it in the benchmark config.

Reply:
Yes why not